### PR TITLE
Change latest stable version to 0.10.0

### PR DESCRIPTION
**Pull request by AGZ98**
This fixes https://github.com/coala/coala/issues/3696
This changes the latest stable version to 0.10.0 (from 0.9.0) in README.rst

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -246,7 +246,7 @@ License
 
 .. |Windows| image:: https://img.shields.io/badge/platform-Windows-brightgreen.svg
 .. |Linux| image:: https://img.shields.io/badge/platform-Linux-brightgreen.svg
-.. |Stable| image:: https://img.shields.io/badge/latest%20stable-0.9.0-green.svg
+.. |Stable| image:: https://img.shields.io/badge/latest%20stable-0.10.0-green.svg
```
Comment (damngamerz) [inline README.rst]: This seems to be good !! You have to follow commit guide lines!
https://api.coala.io/en/latest/Developers/Writing_Good_Commits.html
Also you have 3 different commits for this.While it's a typo change can be done in one single commit.Try soft reset and then stage and commit followed by force push :smiley:

**Comment by all3fox** [comment]
ack b729c2e7dd1490dba864a528a7a16f7732e1fef8

**Comment by meetmangukiya** [comment]
Please squash your commits and also fix the authorship details :)

**Comment by meetmangukiya** [comment]
we don't have a badge now

**Pull request closed without merging**